src/main/java/no/nav/finnkandidatapi/kandidat/KandidatService.java

```diff
@@ -117,6 +131,7 @@ public KandidatService(
             eventPublisher.publishEvent(new KandidatSlettet(slettetKey.get(), oppfølgingAvsluttetMelding.getAktørId(), Brukertype.SYSTEM, dateProvider.now()));
             log.info("Slettet kandidat med id {} pga. avsluttet oppfølging", slettetKey.get());
         }
+        permittertArbeidssokerService.behandleOppfølgingAvsluttet(oppfølgingAvsluttetMelding);
```

Comment:
Trenger man gjøre dette kallet fra KandidatService?
Hadde det blitt mer ryddig om man gjør `permittertArbeidssokerService.behandleOppfølgingAvsluttet` i `OppfølgingAvsluttetConsumer`?

Reply:
Jeg gjorde det sånn først, men så gjorde jeg om på det.. Det er litt ett-fett for meg, jeg kan gjerne endre. Jeg tenkte at iom at den andre eventen går gjennom KandidatService, så kunne jeg kanskje like gjerne gjøre det her også, så PermittertArbeidssokerService alltid lå "under" KandidatService og ikke gjorde noe på egen hånd. Men som sagt, jeg kan gjerne endre! :)